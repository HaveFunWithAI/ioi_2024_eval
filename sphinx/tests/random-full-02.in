250 249
62 98 73 44 30 75 75 151 12 48 75 91 62 214 214 25 107 75 151 189 12 80 91 151 214 148 73 53 42 42 117 30 151 48 53 119 119 151 66 174 107 83 233 157 80 214 213 91 189 30 174 75 233 12 214 44 189 50 62 69 98 62 213 66 119 233 42 42 174 174 189 214 85 42 148 148 66 83 56 80 223 213 48 66 48 213 56 73 12 42 157 25 237 107 73 56 91 213 91 174 107 50 80 189 75 91 73 69 148 91 42 91 189 119 148 56 91 9 119 174 126 91 53 66 197 69 30 53 53 157 69 233 151 42 56 98 62 151 48 197 44 117 50 75 66 80 107 151 214 107 48 80 80 73 75 25 25 148 44 237 50 80 213 148 214 62 117 56 62 44 237 151 83 80 42 126 62 30 30 157 223 75 151 75 66 42 126 73 83 83 30 85 53 91 98 83 66 66 214 30 66 50 107 12 148 75 197 75 53 25 56 91 25 189 197 107 223 233 151 197 62 214 151 42 189 12 80 12 9 107 98 233 50 50 91 119 12 151 223 62 197 62 233 223 126 25 56 117 75 233
203 220
42 220
72 220
51 203
51 146
13 72
72 236
146 231
203 244
213 220
13 143
90 231
51 134
88 203
174 213
143 224
62 134
243 244
29 88
29 189
53 88
146 227
67 90
8 29
51 92
42 187
166 220
8 225
53 207
91 231
7 42
7 130
72 150
21 174
28 231
7 125
28 135
232 244
28 190
92 123
161 231
42 147
220 239
9 146
5 62
48 227
35 125
84 125
29 142
104 225
99 161
90 137
130 233
113 174
17 161
8 248
125 230
142 154
207 215
0 244
0 61
165 244
9 95
205 230
53 192
83 134
137 169
42 101
134 209
30 125
81 154
45 224
130 141
75 137
38 113
83 140
56 81
90 202
51 59
38 76
92 176
24 165
130 237
137 218
5 103
34 56
35 44
113 121
38 39
44 122
0 57
94 142
30 77
39 124
73 224
38 144
3 57
155 227
144 238
63 134
42 193
51 249
27 237
71 84
73 235
98 141
47 81
95 171
18 99
78 121
26 27
229 238
103 153
101 182
27 195
65 238
19 35
66 224
27 119
78 217
107 153
46 122
15 202
103 112
139 236
42 234
55 88
71 145
106 169
68 209
36 244
24 127
147 159
91 184
40 101
6 174
120 146
92 178
52 62
27 129
132 169
114 184
7 31
26 197
131 205
87 104
70 243
67 79
132 152
180 184
122 206
83 102
10 215
60 249
51 115
29 183
93 215
49 59
172 218
45 210
105 112
82 236
113 241
25 155
131 247
1 182
136 187
78 151
1 138
96 146
38 43
61 89
194 247
194 204
111 225
55 74
208 231
77 80
35 37
16 210
38 157
90 221
71 199
101 216
11 94
123 214
22 239
2 35
17 191
48 128
212 230
222 231
93 179
14 81
143 240
94 170
192 223
72 198
203 228
151 163
1 50
110 193
100 155
71 108
29 118
34 164
126 236
20 72
22 173
94 211
39 188
60 167
119 185
97 198
147 246
156 221
196 212
91 117
149 169
52 54
32 217
12 206
143 200
33 97
148 213
41 180
116 121
132 133
56 86
226 241
85 237
215 245
168 170
219 246
58 226
22 177
69 171
64 143
114 160
4 172
23 139
14 109
175 239
35 158
203 242
172 186
201 222
143 162
181 190
