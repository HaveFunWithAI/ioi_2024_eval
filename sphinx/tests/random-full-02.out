250 2535
62 98 73 44 30 75 75 151 12 48 75 91 62 214 214 25 107 75 151 189 12 80 91 151 214 148 73 53 42 42 117 30 151 48 53 119 119 151 66 174 107 83 233 157 80 214 213 91 189 30 174 75 233 12 214 44 189 50 62 69 98 62 213 66 119 233 42 42 174 174 189 214 85 42 148 148 66 83 56 80 223 213 48 66 48 213 56 73 12 42 157 25 237 107 73 56 91 213 91 174 107 50 80 189 75 91 73 69 148 91 42 91 189 119 148 56 91 9 119 174 126 91 53 66 197 69 30 53 53 157 69 233 151 42 56 98 62 151 48 197 44 117 50 75 66 80 107 151 214 107 48 80 80 73 75 25 25 148 44 237 50 80 213 148 214 62 117 56 62 44 237 151 83 80 42 126 62 30 30 157 223 75 151 75 66 42 126 73 83 83 30 85 53 91 98 83 66 66 214 30 66 50 107 12 148 75 197 75 53 25 56 91 25 189 197 107 223 233 151 197 62 214 151 42 189 12 80 12 9 107 98 233 50 50 91 119 12 151 223 62 197 62 233 223 126 25 56 117 75 233
